src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -40,6 +40,20 @@ public boolean insert(String tableName, DataSet dataset) {
         }
     }
 
+    /*
+     * Method for deleting data from a table.
+     * If successful, the method returns true, else return false
+     */
+    public boolean delete(String tableName, int id) {
+        try (Statement statement = connection.createStatement()) {
+            statement.execute(String.format("DELETE FROM %s WHERE id = %s", tableName, id));
```

Comment:
If table name is correct then this line will be executed for any value of id and method will return true, but according to the task if id not exists method shoul return false.

Reply:
Ok, i will do it.